ikarus/python/dirichletvalues/dirichletvalues.hh

```diff
@@ -29,13 +31,15 @@ namespace Ikarus::Python {
  * This function registers Python bindings for a DirichletValues class, allowing it to be used in Python scripts.
  * The registered class will have an initializer that takes a `Basis` object. It exposes several member functions to
  * Python:
- *   - `fixBoundaryDOFs(f)`: Fixes boundary degrees of freedom using a user-defined function `f`.
- *   - `fixBoundaryDOFsUsingLocalView(f)`: Fixes boundary degrees of freedom using a user-defined function `f` with a
- * `LocalView` argument.
- *   - `fixBoundaryDOFsUsingLocalViewAndIntersection(f)`: Fixes boundary degrees of freedom using a user-defined
- * function `f` with `LocalView` and `Intersection` arguments.
- *   - `fixDOFs(f)`: Fixes boundary degrees of freedom using a user-defined function `f` with the boolean vector and
+ *   - `fixBoundaryDOFs(f)`: Fixes boundary degrees of freedom. This function can be overloaded with the following
+ * arguments:
+ *    - using a user-defined function `f`.
```

Comment:
These argument descriptions are not correct.

The arguments are also the boolean vector and the local indices or global indices.